public/app/percona/dbaas/components/DBCluster/DBCluster.hooks.ts

```diff
@@ -59,7 +59,10 @@ const getClusters = async (kubernetes: Kubernetes[], databaseType: Databases): P
   const kubernetesByOperator = kubernetes.filter(kubernetesCluster => {
     const operator = OPERATORS[databaseType] as Operators;
 
-    return kubernetesCluster.operators[operator].status === KubernetesOperatorStatus.ok;
+    return (
+      kubernetesCluster.operators[operator].status === KubernetesOperatorStatus.ok ||
+      kubernetesCluster.operators[operator].status === KubernetesOperatorStatus.unsupported
+    );
```

Comment:
consider doing something like:

```suggestion
    const operatorStatus = kubernetesCluster.operators[operator].status;
    
    return (
      operatorStatus === KubernetesOperatorStatus.ok || operatorStatus === KubernetesOperatorStatus.unsupported
    );
```

Reply:
Thanks @nicolalamacchia, done!